src/sidebar/components/thread-card.js

```diff
@@ -0,0 +1,65 @@
+import classnames from 'classnames';
+import { createElement } from 'preact';
+import { useCallback, useMemo } from 'preact/hooks';
+
+import debounce from 'lodash.debounce';
+import propTypes from 'prop-types';
+import useStore from '../store/use-store';
+import { withServices } from '../util/service-context';
+
+import Thread from './thread';
+
+/**
+ * A "top-level" `Thread`, rendered as a "card" in the sidebar. A `Thread`
+ * renders its own child `Thread`s within itself.
+ */
+function ThreadCard({ frameSync, settings, thread }) {
+  const threadTag = useMemo(() => {
+    return thread.annotation && thread.annotation.$tag;
+  }, [thread]);
+
+  const isFocused = useStore(store => store.isAnnotationFocused(threadTag));
+
+  const focusThreadAnnotation = useCallback(
+    debounce(tag => {
+      const focusTags = tag ? [tag] : [];
+      frameSync.focusAnnotations(focusTags);
+    }, 10),
+    [frameSync]
+  );
+
+  const scrollToAnnotation = useCallback(
+    tag => {
+      frameSync.scrollToAnnotation(tag);
+    },
+    [frameSync]
+  );
+
+  return (
+    /* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */
+    <div
+      onClick={() => scrollToAnnotation(threadTag)}
```

Comment:
This makes the a11y linter sad. It's a reimplementation of existing functionality. I don't know if there's a keyboard event that could be mapped to this in a meaningful way?

Reply:
>  I don't know if there's a keyboard event that could be mapped to this in a meaningful way?

I suppose the closest equivalent to hover for the keyboard would be focus being within a particular region, so perhaps `focusin` and `focusout`. That's a change in behaviour I don't think we should try to cram into this PR though.